adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/internal/service/ParametersValidationService.java

```diff
@@ -0,0 +1,42 @@
+package de.adorsys.xs2a.adapter.service.ing.internal.service;
+
+import de.adorsys.xs2a.adapter.service.Oauth2Service;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParametersValidationService {
+
+    private static final String SCOPE = Oauth2Service.Parameters.SCOPE;
+    private static final String DEFAULT_SCOPE = "payment-accounts:transactions:view";
+    private static final List<String> availableScopes = Arrays.asList(DEFAULT_SCOPE, "payment-accounts:balances:view");
+
+    public static Oauth2Service.Parameters validateScope(Oauth2Service.Parameters params) {
+        if (params.get(SCOPE) == null) {
```

Comment:
i suggest to use` StringUtils.isBlank` for checking

Reply:
done